samples/sensor/magn_polling/sample.yaml

```diff
@@ -11,3 +11,4 @@ tests:
       - sensortile_box          # lis2mdl
       - stm32f411e_disco        # lsm303agr_magn
       - stm32f3_disco           # lsm303dlhc_magn
+      - rddrone_fmuk66          # bmm150
```

Comment:
We already have an integration platform for this sensor, so we don't need to add another one

Reply:
Acknowledged